kivy/uix/textinput.py

```diff
@@ -2583,6 +2585,15 @@ def keyboard_on_textinput(self, window, text):
         if self._selection:
             self.delete_selection()
         self.insert_text(text, False)
+        self._is_textedit=False #准备插入，对束edit状态
+ 
+    def keyboard_on_textedit(self, window, text):
+        #print('textedit',text)
+        if len(text)>0:
```

Comment:
```suggestion
        if len(text) > 0:
```
pep8 spaces around operator.